### DISCUSSION TITLE
Power Transfer Distribution Factors

### DISCUSSION THREAD

**Discussion by realmarcelfried** [e2nIEE/pandapower#1074] [Q&A]
Hey,

I am trying to simulate flow-based market clearing. For that I need the Power Transfer Distrubution Factors (PTDF´s) of my grid. 

Is it possible to do that with the pandapower module?

**Comment by wangzhenassd** [comment]
Hi,
you can try to perturb directly with dc powerflow or access the internal architecture and calculate the PTDF directly with the pypower functionality (see also #847, #784)

**Reply by Spatran** [reply]
Hi,
Did you use PSAT Toolbox for designing PTDF?

**Reply by rbolgaryn** [reply]
Hi @Spartan, no we haven't. The PTDF fonctionality in pandapower comes from pypower. What are the advantages/features in PSAT Toolbox for PTDF?
